### ISSUE TITLE
Bulleted lists aren't rendering properly in parts of analysis commons chapter

### ISSUE THREAD

**Issue by emeryl** [UW-GAC/topmed_workshop_2017#5]
These sections in the analysis commons worksheet chapter don't render bulleted lists properly in the bookdown document. Since markdown documents are always rendered as html, I can't link to the affected line numbers, but below are links to the sections at issue. Note that the lists render appropriately on GitHub, but not in the bookdown document.
@smgogarten @j-brody 

https://uw-gac.github.io/topmed_workshop_2017/analysis-commons.html#web-interface-and-running-an-analysis-application

https://uw-gac.github.io/topmed_workshop_2017/analysis-commons.html#command-line-interface

**Comment by j-brody** [comment]
Hi Leslie,

	I’m a complete markdown/bookdown novice.  I tried changing the syntax and it might work now.  If not, any suggestions would be welcome.  

Thanks!

**Comment by j-brody** [comment]
Hi Leslie,

	I just noticed a couple of other things that had not rendered correctly.  I just updated the markdown on GitHub again.  

-Thanks, Jen

**Comment by emeryl** [comment]
Hi Jen,

I just looked into this more and the bookdown document still wasn't showing these bulleted lists properly. I realized that you probably can't test it very easily though, since we're using _build.sh on one of our GAC servers to render the bookdown document. So I fiddled around with the file a bit until the lists looked good. There were a couple of things going on:

- Not sure if this was an issue, but I changed the "-"s to "*"s for the bullets, since that's the standard for pandoc markdown (which R markdown is built on)
- I put slashes in to escape all of the underscores in filenames, since underscores are also used to indicate italics
- I added two spaces at the end of every line in the bulleted lists and the line beforehand

Take a look at the newly rendered chapter 9 and let me know if it looks okay to you.
